src/config/select.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
Good question. I just checked and it seems like this is not the case yet.

<img width="1000" alt="screen shot 2015-10-12 at 10 17 17" src="https://cloud.githubusercontent.com/assets/223045/10423851/79ada2aa-70ca-11e5-9bac-ff9b4e342cfb.png">